src/random_number_generator.cc

```diff
@@ -57,7 +57,54 @@ namespace cyclus{
             val = rn();
         }
         int rounded_val = std::lrint(val);
+        return rounded_val;
+    }
+
+    //
+    // Distributions
+    UniformDoubleDist::UniformDoubleDist(double min, double max) : dist(min, max) {}
+
+    double UniformDoubleDist::sample() { 
+        return UniformDoubleDist::dist(RandomNumberGenerator::gen_); 
```

Comment:
Shouldn't this just be:

```suggestion
        return dist(RandomNumberGenerator::gen_); 
```

since `dist` is a member variable of this particular object?